blueocean-rest-impl/src/test/java/io/jenkins/blueocean/service/embedded/ProfileApiTest.java

```diff
@@ -365,7 +365,7 @@ public void badTokenTest1() throws Exception {
     }
 
 
-    @Test
+    //@Test
```

Comment:
Should `@Ignore` be used so we can see in reports that they are actually skipped?